src/appengine/handlers/cron/project_setup.py

```diff
@@ -193,7 +193,7 @@ def get_github_url(url):
   response = requests.get(url, auth=(client_id, client_secret))
   if response.status_code != 200:
     logs.log_error(
-        'Failed to get github url: %s' % url, status_code=response.status_code)
+        f'Failed to get github url: {url}.', status_code=response.status_code)
```

Comment:
Sometimes it's good to fix code when you see it can be improved but often times it's easier for reviewers if PRs cover one major change at a time (e.g. adding a feature) and not contain other smaller changes (such as this: changing format strings).

Reply:
Yep, thanks.
I probably should split Commit [format and lint](https://github.com/google/clusterfuzz/pull/2578/commits/eb09218715f6fc177545bfbfde7e84802f818d3c) to an independent PR.

Reply:
The linter didn't ask you to make these changes did it?
If it did, then you should do it in this PR (IMO it's a bug in the way we lint if it did tell you this)

Reply:
I wouldn't bother splitting anything out now. Just for future reference.

Reply:
The local lint (`python bulter.py lint`) did not, but the lint from CI did.
That is how I noticed them : )